dev-aws/kafka-shared-msk/data-infra/data-infra.tf

```diff
@@ -49,6 +49,24 @@ resource "kafka_topic" "snowplow" {
   }
 }
 
+resource "kafka_topic" "test_pla1275" {
+  name               = "data-infra.uw.data-infra.test.pla1275"
+  replication_factor = 1
+  partitions         = 1
+  config = {
+    # this is a test, and we need minimum and non-durable resources
+    "remote.storage.enable" = "false"
+    # 1 day
+    "retention.ms" = "86400000"
+    # keep data in hot storage for 1 day
+    # "local.retention.ms" = "86400000"
```

Comment:
please remove this line and the comment above

Reply:
addressed